pre-commit

```diff
@@ -53,3 +53,5 @@ cargo clippy --tests --features shuttle -- -D warnings || error "Clippy concurre
 cargo test || error "Test failures"
 
 cargo test --features shuttle || error "Concurrency test failures"
+
+cargo bench --bench oneshot_ipa --no-default-features --features="enable-benches" || error "Failed to run IPA benchmark"
```

Comment:
Later, should we add an argument for batchsize and run this only for 10 rows in pre-commit. We are slowly increasing the pre-commit time more and more